packages/css/src/unordered-list/unordered-list.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-unordered-list {
+  color: var(--amsterdam-unordered-list-color);
+  font-family: var(--amsterdam-unordered-list-font-family);
+  font-size: var(--amsterdam-unordered-list-font-size-narrow);
+  font-weight: var(--amsterdam-unordered-list-font-weight);
+  line-height: var(--amsterdam-unordered-list-line-height);
+  list-style: var(--amsterdam-unordered-list-list-style);
+  margin-block: var(--amsterdam-unordered-list-margin-block);
+  padding-inline-end: var(--amsterdam-unordered-list-padding-inline-end);
```

Comment:
Do we need this?

Reply:
Also something from Figma

Reply:
So box sizing to realise the design? If you need to add `ul` styling to every div that can contain it you'll create a lot of rubish.

Reply:
Not to mention the places that you won't think of where developers can use this component.